### ISSUE TITLE
I feel like citra mmj doesn't like water

### ISSUE THREAD

**Issue by Ahmedmksd** [weihuoya/citra#347]
You think I'm joking? I'm serious! almost every game with water in it has graphical issues with simulating water like Kirby planet robobot, Mario and Luigi bowser inside story and superstar saga, and I'm pretty sure that everyone knows how pokemon oras and xy work with water types (even though it works with accurate multiple accurate,but it kills performance)oh and one extra unrelated problem, infinite loading screen with mario in ssb4 3ds

**Comment by Ahmedmksd** [comment]
Oh and i can't forget Luigi mansion 2, every time i open it it just does a big smelly in every corner

**Comment by TheOG00** [comment]
Yeah Luigi mansion 2 is not playable & mario 3d land still freezing a bit

**Comment by SmileFacey** [comment]
So the water and lava issues still aren't fixed on Citra MMJ?
Even with Accurate Nul?

**Comment by Ahmedmksd** [comment]
> So the water and lava issues still aren't fixed on Citra MMJ?
> Even with Accurate Nul?

Yes

**Comment by Ahmedmksd** [comment]
> Yeah Luigi mansion 2 is not playable & mario 3d land still freezing a bit

Tbh mario 3d land is just loading and it's something I can live with

**Comment by CyberBlueCodes** [comment]
It works on mine without acurrute shamser mul

**Comment by CyberBlueCodes** [comment]
And also can anyone answer #359

**Comment by Ahmedmksd** [comment]
> It works on mine without acurrute shader mul

How

**Comment by Uder2** [comment]
My very good new super Mario 3ds gane freezes at the end of game
